.openpublishing.redirection.json

```diff
@@ -195,18 +189,193 @@
         "redirect_document_id": true
     },
     {
+        "source_path": "docs/core/preview3/tutorials/using-with-xplat-cli-msbuild-folders.md", 
+        "redirect_url": "/dotnet/articles/core/tutorials/testing-with-cli",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs/core/preview3/windows-prerequisites.md",
+        "redirect_url": "/dotnet/articles/core/windows-prerequisites"
+    },
+    {
+        "source_path": "docs/core/testing/using-mstest-on-windows.md", 
```

Comment:
Great catch! I fixed this in the second commit -  "Added redirect_document_id=true per review comment".